pagination/src/main/java/com/cerner/beadledom/pagination/OffsetPaginatedList.java

```diff
@@ -0,0 +1,94 @@
+package com.cerner.beadledom.pagination;
+
+import com.cerner.beadledom.pagination.parameters.LimitParameter;
+import com.cerner.beadledom.pagination.parameters.OffsetParameter;
+
```

Comment:
Looks like your formatter is not following the latest google style guide which no longer groups imports.

Reply:
Here is the updated formatter, https://github.cerner.com/hi-infra/appservices-styleguide/tree/updated-google-style

Reply:
Updated with the latest https://plugins.jetbrains.com/plugin/8527-google-java-format , let me know if that was what you were looking for

Reply:
Didn't know about the plugin, awesome! It still doesn't do import order correctly though, https://github.com/google/google-java-format#intellij 

```
The import ordering is not handled by this plugin, unfortunately. To fix the import order, download the IntelliJ Java Google Style file and import it into File→Settings→Editor→Code Style.
```

I installed the plugin and the Google code style, and everything seems good now. Thanks for the info @ikottman

Reply:
Whoops, totally missed that. I updated my checkstyle rules and re-optimized imports, _now_ the imports should be correct.